connmgr/null.go

```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/libp2p/go-libp2p-core/peer"
 )
 
+// NullConnMgr is a ConnMgr that provides no functionality. It is used by go-libp2p-blankhost
+// to satisfy the ConnMgr interface.
```

Comment:
The forward reference to blankhost might be unnecessary. Also, it is the default if we don't specify a `ConnManager` option in the libp2p constructor.